stl/inc/xatomic.h

```diff
@@ -9,7 +9,12 @@
 #include <yvals.h>
 #if _STL_COMPILER_PREPROCESSOR
 
+// TRANSITION, Visual Studio 2019 version 16.8
+#if defined(_MSC_VER) && _MSC_VER > 1927

```

Comment:
I don't think this is going to work - it will immediately activate for VS 2019 16.8 Preview 1. Should we simply include `intrin0.h` and manually declare the extra intrinsics as a workaround, like @barcharcraz did for `<bit>`?

Reply:
@BillyONeal did this way, I'm not sure.

Reply:
>manually declare the extra intrinsics as a workaround, like @barcharcraz did for <bit>?

I don't think we should have done that. Can we check `_MSC_FULL_VER` for whatever the preview 2 version is?

Reply:
@StephanTLavavej Because every time folks try to declare intrinsics themselves it leads to trouble. Look at that Windows 8.1 SDK bug that happened recently because the Windows SDK declares some of the intrinsics with a slightly  different signature that causes trouble.

It's less about intrinsics specifically and more about there being one canonical source of truth for any declaration; I view declaring `intrin.h` stuff as morally the same as our customers forward declaring stuff in `std::`.

Reply:
I made it `#if 1` then instead of version check. Will remove manually in a subsequent change.

Reply:
I asked Cameron and he says the 16.8p2 `_MSC_FULL_VER` should be 192829203 so we should be able to test against that.